script.js

```diff
@@ -48,3 +48,66 @@ const photos = [
         type: 'animals',
     },
 ];
+
+
+// Homework
+let gallerySpace = document.querySelector("#gallery");
+
+let allButton = document.querySelector("#all");
+let natureButton = document.querySelector("#nature");
+let cityButton = document.querySelector("#city");
+let animalButton = document.querySelector("#animals");
+
+let loadImage = document.querySelector("#loadMore");
+let limit = 0;
+
+const images = (imgType) => {
+
+    let filteredImages = [];
+    gallerySpace.innerHTML = "";
+
+    if (imgType == "all"){
+        filteredImages = photos;
+    }
+    else{
+        photos.forEach((img) => {
+            if (img.type == imgType){
+                filteredImages.push(img);
+            }  
+        });
+    }
+    
+    if(filteredImages.length > 6){
+        for (let img = 0; img < 6; img++){
+            gallerySpace.innerHTML += `<img src=${filteredImages[img].url}>`;
+        }
+        loadImage.addEventListener("click", () => {
+            if (limit < filteredImages.length){
```

Comment:
You should handle when all the available images are shown, then no more photos should be added and the "Load more" button should be hidden.